src/main/java/org/spongepowered/api/world/gen/WorldGenerator.java

```diff
@@ -35,16 +32,17 @@
 public interface WorldGenerator {
 
     /**
-     * Generates the chunk at the given position in the world. The position
-     * specified is the chunk position (ie. the world position divided by the
-     * chunk size). The newly generated chunk should be placed into the given
-     * {@link BlockBuffer}.
-     *
-     * @param world The world
-     * @param buffer The buffer to generate the region into
-     * @param position The chunk position
+     * Gets the {@link GeneratorPopulator} of this world generator.
+     * @return The {@link GeneratorPopulator}.
+     * @see #setGeneratorPopulator(GeneratorPopulator)
+     */
+    GeneratorPopulator getGeneratorPopulator();
```

Comment:
As said elsewhere, the impl bends to the API's will, always. This needs to be a list.